src/App.js

```diff
@@ -1,10 +1,17 @@
 import logo from './logo.svg';
 import './App.css';
+import Counter from './mini_components/functional'
+import HelloComp from './mini_components/react_component';
+
+const name = 'Vinodel'
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
+        <div>this is React.component</div>
+        <HelloComp name={name} />
```

Comment:
Consider using props inside of your component when you do actually pass them from parent.